README.adoc

```diff
@@ -33,6 +35,32 @@ Car ferrari = Car.id(10L).name("Ferrari").type(CarType.SPORT);
 repository.save(ferrari);
 ----
 
+* *Custom Repository Interfaces*: Besides the built-in repository interfaces, Jakarta Data enables you to create custom repository interfaces tailored to your domain requirements. These custom interfaces serve as a bridge between your domain's semantics and the underlying database operations. With annotations like `Insert`, `Update`, `Delete`, and `Save`, you can define expressive and domain-specific repository methods that align seamlessly with your application's needs.
+
+[source,java]
+----
+@Repository
+public interface Garage {
+
+  @Save
+  Car park(Car car);
```

Comment:
`Insert` is better aligned with `Save` for a method that is named `park`.  It doesn't make sense to refer to updating the state of a Car (maybe its license plate number) as parking it.

```suggestion
  @Insert
  Car park(Car car);
```